### PR TITLE
Dotnet zstream 4.16 and firewatch updates

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.15-lp-interop-dotnet-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.15-lp-interop-dotnet-interop-ibmcloud periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.16-lp-interop-dotnet-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.16-lp-interop-dotnet-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.15-lp-interop-dotnet-interop-ibmcloud periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.16-lp-interop-dotnet-interop-ibmcloud

File: ci-operator/jobs/sclorg/ansible-tests/sclorg-ansible-tests-master-periodics.yaml [hunk 1]
```diff
@@ -535,6 +534,86 @@ periodics:
...
+  - base_ref: master
+    org: sclorg
+    repo: ansible-tests
+  labels:
+    ci-operator.openshift.io/cloud: ibmcloud
+    ci-operator.openshift.io/cloud-cluster-profile: ibmcloud-cspi-qe
+    ci-operator.openshift.io/variant: dotnet-ocp4.16-lp-interop
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-sclorg-ansible-tests-master-dotnet-ocp4.16-lp-interop-dotnet-interop-ibmcloud
```
Comment (calebevans) [inline ci-operator/jobs/sclorg/ansible-tests/sclorg-ansible-tests-master-periodics.yaml:553]: Slack config

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**